modules/nw-sriov-configuring-operator.adoc

```diff
@@ -208,3 +254,48 @@ spec:
     <node_label>
 ----
 ====
+
+[id="configure-sr-iov-operator-single-node_{context}"]
+== Configuring the SR-IOV Network Operator for single node installations
+
+By default, the SR-IOV Network Operator drains workloads from a node before every policy change.
+The Operator performs this action to ensure that there no workloads using the virtual functions before the reconfiguration.
+
+For installations on a single node, there are no other nodes to receive the workloads.
+As a result, the Operator must be configured not to drain the workloads from the single node.
+
+After performing the following procedure to disable draining workloads, you must remove any workload that uses an SR-IOV network interface before you change any SR-IOV network node policy.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* You must have installed the SR-IOV Network Operator.
+
+.Procedure
+
+- To set the `disableDrain` field to `true`, enter the following command:
```

Comment:
```suggestion
* To set the `disableDrain` field to `true`, enter the following command:
```

Super duper nit picky thing, use of * instead of - for bullet.

Reply:
@sagidlow , the rest of the file uses a hyphen for the bullets.  I can make a follow up PR.